includes/Story_Post_Type.php

```diff
@@ -519,7 +519,11 @@ public static function admin_body_class( $class ) {
 			return $class;
 		}
 
-		$class .= ' edit-story ';
+		$class .= ' edit-story';
+
+		if ( false === strpos( $class, 'folded' ) ) {
```

Comment:
Let's leave a comment here to explain the purpose of this for future readers.
E.g. "Reverse the default WP behavior by folding the menu by default" or sth like that.

Reply:
Sounds good. Will add'n'merge.